### PR TITLE
Edit status check

### PR DESCRIPTION
**Pull request by jackwaudby**
Minor improvements to the status check page, mainly aligning the formatting closer to the other monitoring pages:
- Using tables for the arguments and return values 
- Change title to 'Monitor replication status'
- Add an example of sample output

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1836.surge.sh

Updated pages:
- [Monitor replication status](https://neo4j-docs-operations-1836.surge.sh/operations-manual/5/clustering/monitoring/status-check)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1836.surge.sh

Updated pages:
- [Clustering](https://neo4j-docs-operations-1836.surge.sh/operations-manual/5/clustering)
- [Monitor replication status](https://neo4j-docs-operations-1836.surge.sh/operations-manual/5/clustering/monitoring/status-check)

File: modules/ROOT/pages/clustering/monitoring/status-check.adoc [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+[[monitoring-replication]]
+= Monitor replication status
 
-Neo4j 5.24 introduces the xref:reference/procedures.adoc#procedure_dbms_cluster_statusCheck[`dbms.cluster.statusCheck()`] procedure, which can be used to monitor the ability to replicate in clustered databases, which in most cases means being able to write to the database.
-You can also use the procedure to check which members are up-to-date and can participate in a successful replication.
-Therefore, it is useful in determining the fault-tolerance of a clustered database as well.
-A third and final function is to determine the leader of the cluster.
+Neo4j 5.24 introduces the xref:reference/procedures.adoc#procedure_dbms_cluster_statusCheck[`dbms.cluster.statusCheck()`] procedure, which can be used to monitor the ability to replicate in clustered databases. 
+In most cases this means a clustered database is write available.
+The procedure can also be used to check which members of a clustered database are up-to-date and can participate in a successful replication.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/monitoring/status-check.adoc:9]:
```suggestion
The procedure identifies which members of a clustered database are up-to-date and can participate in successful replication.
```

File: modules/ROOT/pages/clustering/monitoring/status-check.adoc [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+Therefore, it is useful in determining the fault-tolerance of a clustered database.
+A third and final function is to determine which member is the leader of a clustered database within the cluster.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/monitoring/status-check.adoc:11]:
```suggestion
Additionally, you can use the procedure to identify the leader of a clustered database within the cluster.
```

File: modules/ROOT/pages/clustering/monitoring/status-check.adoc [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
+= Monitor replication status
 
-Neo4j 5.24 introduces the xref:reference/procedures.adoc#procedure_dbms_cluster_statusCheck[`dbms.cluster.statusCheck()`] procedure, which can be used to monitor the ability to replicate in clustered databases, which in most cases means being able to write to the database.
-You can also use the procedure to check which members are up-to-date and can participate in a successful replication.
-Therefore, it is useful in determining the fault-tolerance of a clustered database as well.
-A third and final function is to determine the leader of the cluster.
+Neo4j 5.24 introduces the xref:reference/procedures.adoc#procedure_dbms_cluster_statusCheck[`dbms.cluster.statusCheck()`] procedure, which can be used to monitor the ability to replicate in clustered databases. 
+In most cases this means a clustered database is write available.
+The procedure can also be used to check which members of a clustered database are up-to-date and can participate in a successful replication.
+Therefore, it is useful in determining the fault-tolerance of a clustered database.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/monitoring/status-check.adoc:10]:
```suggestion
Therefore, it is useful in determining the fault tolerance of a clustered database.
```

File: modules/ROOT/pages/clustering/monitoring/status-check.adoc [hunk 4]
```diff
@@ -18,45 +19,62 @@ Apart from replication there are other stops in the write path that can potentia
...
+| requester             | Boolean      | Whether a server is the requester or not.
+| error                 | String       | Contains the error message if one is present. 
+An example of an error is that one or more of the requested databases do not exist on the requester.
+|===
+
+=== Possible values of `replicationSuccessful`
+* `TRUE` -- if this server managed to replicate the dummy transaction to a majority of cluster members within the given timeout.
+* `FALSE` -- if it failed to replicate within the timeout.
 The value is the same column-wise.
 A failed replication can either mean a real issue in the cluster (e.g., no leader) or that this server is too far behind in apply and can't replicate.
```
[the remark this answers is not in the document]
Comment (jackwaudby) [reply modules/ROOT/pages/clustering/monitoring/status-check.adoc:67]: 'too far behind in applying updates' is also ok

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1836.surge.sh

Updated pages:
- [Clustering](https://neo4j-docs-operations-1836.surge.sh/operations-manual/5/clustering)
- [Monitor replication status](https://neo4j-docs-operations-1836.surge.sh/operations-manual/5/clustering/monitoring/status-check)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1836.surge.sh

Updated pages:
- [Clustering](https://neo4j-docs-operations-1836.surge.sh/operations-manual/5/clustering)
- [Monitor replication status](https://neo4j-docs-operations-1836.surge.sh/operations-manual/5/clustering/monitoring/status-check)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1836.surge.sh

Updated pages:
- [Clustering](https://neo4j-docs-operations-1836.surge.sh/operations-manual/5/clustering)
- [Monitor replication status](https://neo4j-docs-operations-1836.surge.sh/operations-manual/5/clustering/monitoring/status-check)

**Review by NataliaIvakina — APPROVED** [review]
LGTM!

**Pull request merged**